helm/oauth2-proxy/templates/servicemonitor.yaml

```diff
@@ -9,20 +9,18 @@ metadata:
   namespace: {{ .Release.Namespace | quote }}
 {{- end }}
   labels:
-    app: {{ template "oauth2-proxy.name" . }}
-    chart: {{ template "oauth2-proxy.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+  labels:
     prometheus: {{ .Values.metrics.servicemonitor.prometheusInstance }}
+    app: {{ template "oauth2-proxy.name" . }}
```

Comment:
Is this one needed?

Reply:
this same additional label is on the service, too - kept it for consistency